packages/algob/src/types.ts

```diff
@@ -311,6 +311,7 @@ export interface MultiSig {
 }
 
 export interface LogicSig {
+  l: Uint8Array
   logic: Uint8Array
   args: Object
```

Comment:
what is the difference between `l` and `logic` ?

Reply:
Both represent `logic` part of the logic signature.

Reply:
we should use only one, as this seems confusing for other developers

Reply:
Continued on discord.